docs/content/tutorial/setup.mdx

```diff
@@ -9,7 +9,7 @@ To complete this tutorial, you'll need:
 
 - **To install Python and pip.** This tutorial assumes that you have some familiarity with Python, but you should be able to follow along even if you're coming from a different programming language. To check that Python and `pip` (Python's package manager) are already installed in your environment or install them, follow the instructions [here](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/).
 
-  {" "} <DagsterVersion />
+<DagsterVersion />
```

Comment:
🔧 This should be indented to be in line with the list item that contains it. I wonder if `mdx-format` is throwing a fit about this because it's a component?

Reply:
Remember when you asked how you can help? Do you mind helping me wrestle this? 🥺

Reply:
Absolutely :)

Reply:
Done & pushed. The fix was dumb. 🤦‍♀️